scripts/initialize-build-host.sh

```diff
@@ -48,6 +48,13 @@ then
     sed -i -e 's/^\( *Defaults *requiretty *\)$/# \1/' /etc/sudoers
 fi
 
+# Add 512MB swap space, java needs it, and jenkins stops complaining.
+# This will happen on both hosts, if there is a proxy and a build host.
+dd if=/dev/zero of=/swapfile bs=1M count=512
```

Comment:
Here's the whole list of what I did: https://github.com/mendersoftware/mender-qa/compare/5077e69633663fd8da8c228a63e984fb59c3ed2f...c9ad9b19adcc5fbb1ddf233a84a8eced8d3d3245